problem_2/spec/even_fibonacci_numbers_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'even_fibonacci_numbers'
+
+describe FibonacciCalculator do
+	let (:calculator) { FibonacciCalculator.new(limit) }
+		
+	describe '.add_fib_sequence' do
+			
+		context 'when limit is 100' do
```

Comment:
don't think you need `context` since the `let(:limit)` explains that limit is 100

Reply:
I tried taking out the `context` blocks but then the last `let(:limit)` gets applied to every test. Is there a way to pass separate limits without putting it in a `context`?

Reply:
I wouldn't use `let` for these tests. It adds some unnecessary complexity since you're setting `limit` in each test you could write `calculator = FibonacciCalculator.new(100)`